ui/lib/core/addon/components/model-wrap.js

```diff
@@ -0,0 +1,23 @@
+import { inject as service } from '@ember/service';
```

Comment:
a comment would be good here explaining that this is a wrapper so we can pass methods to be called in